packages/style-loader/index.js

```diff
@@ -36,22 +24,28 @@ module.exports = (neutrino, opts = {}) => {
       }
     }
   }, opts);
-
-  const rules = [options];
+  const rules = [];
 
   if (options.modules) {
-    rules.push(merge(options, {
-      test: options.modulesTest,
-      ruleId: `${options.ruleId}${options.modulesSuffix}`,
-      styleUseId: `${options.styleUseId}${options.modulesSuffix}`,
-      cssUseId: `${options.cssUseId}${options.modulesSuffix}`,
-      hotUseId: `${options.hotUseId}${options.modulesSuffix}`,
-      extractId: `${options.extractId}${options.modulesSuffix}`,
-      css: {
-        modules: options.modules,
-        importLoaders: 1
-      }
-    }));
+    rules.push(
+      merge(options, {
```

Comment:
This will add a whole new rule, making it 3 rules long instead of 2. Is that right?

Reply:
Oh nevermind, I see we aren't adding `options` to the original rules array now.

Reply:
Could this be part of the original object for options?

```js
extract: opts.modules && opts.modulesTest || neutrino.regexFromExtensions(['module.css'])
```

May not read the best though. Thoughts?

Reply:
You mean `modulesTest`, not `extract`, right?

Reply:
oh no - you prob mean `exclude`

Reply:
I don't think that would work, as we _only_ want to exclude the modulesTest for the non-modules rule.